content/ember-released-4-3.md

```diff
@@ -32,7 +32,10 @@ Ember.js 4.3 is an incremental, backwards compatible release of Ember with bug f
 
 #### Bug Fixes
 
-Ember.js 4.3 introduced 0 bug fixes.
+Ember.js 4.3 introduced 3 bug fixes.
+* Memory leak in the Router Service class is fixed([#20025](https://github.com/emberjs/ember.js/pull/20025)). This affected both tests and FastBoot, where we construct new instances of the service whenever we call the `visit` API.
+* Don’t serialize default Query Params on RouterService ([#19971](https://github.com/emberjs/ember.js/pull/19971)).  Using `RouterService#transitionTo` does no longer add unspecified query parameters to the URL.
+* Correctly associate props with factory and owner in FactoryManager ([#20024](https://github.com/emberjs/ember.js/pull/20024))
```

Comment:
This will make the linter happy:

```suggestion
Ember.js 4.3 introduced 3 bug fixes.
- Memory leak in the Router Service class is fixed([#20025](https://github.com/emberjs/ember.js/pull/20025)). This affected both tests and FastBoot, where we construct new instances of the service whenever we call the `visit` API.
- Don’t serialize default Query Params on RouterService ([#19971](https://github.com/emberjs/ember.js/pull/19971)).  Using `RouterService#transitionTo` does no longer add unspecified query parameters to the URL.
- Correctly associate props with factory and owner in FactoryManager ([#20024](https://github.com/emberjs/ember.js/pull/20024))
```